src/Data/Vec/Properties.agda

```diff
@@ -509,6 +520,11 @@ lookup-splitAt (suc m) (x ∷ xs) ys (suc i) = trans
   (lookup-splitAt m xs ys i)
   (sym ([,]-map (Fin.splitAt m i)))
 
+toList-++ : (xs : Vec A n)(ys : Vec A m) →
```

Comment:
Very minor, but here and in the proofs below, can you add spaces in between the arguments, i.e. `(xs : Vec A n) (ys : Vec A m)` rather than `(xs : Vec A n)(ys : Vec A m)`